common/djangoapps/student/forms.py

```diff
@@ -95,6 +100,38 @@ def save(self,  # pylint: disable=arguments-differ
             send_password_reset_email_for_user(user, request)
 
 
+class PasswordResetFormForSecondaryEmailNoActive(PasswordResetFormNoActive):
+    error_messages = {
+        'unknown': _(
+            'That e-mail address doesn\'t have an associated user account. Are you sure you\'ve registered a '
```

Comment:
We can improve wording like:

`Are you sure you had added a verified secondary email address for account recovery in your account settings?`
Also
`e-mail` -> `secondary email`